extensions/cornerstone-dicom-sr/src/viewports/OHIFCornerstoneSRViewport.tsx

```diff
@@ -87,9 +92,11 @@ function OHIFCornerstoneSRViewport(props) {
         { servicesManager, extensionManager },
         displaySetInstanceUID
       );
-      const displaySets = displaySetService.getDisplaySetsForSeries(
-        SeriesInstanceUIDs[0]
-      );
+      const displaySets =
+        (srDisplaySet.keyImageDisplaySet && [
```

Comment:
it is werid that you have a `keyImageDisplaySet` proeprty on the `DisplaySet` itself. Is it really a displaySet inside another displaySet?

Reply:
The display set is registered in the normal way, just that it gets created by the viewport, so it needs some sort of refernce to it.